distributed/distributed.yaml

```diff
@@ -310,7 +310,9 @@ distributed:
       interval: 500ms
       disk: true  # Monitor host-wide disk I/O
       host-cpu: false  # Monitor host-wide CPU usage, with very granular breakdown
-      gil-contention: false  # Monitor GIL contention
+      gil-contention: 
```

Comment:
It's unrelated to the specific change being added here, but thoughts on just `gil` instead of `gil-contention`?

Reply:
That's fine with me (https://github.com/dask/distributed/pull/7650/commits/de2f2489d252dfa804020bb249a9282dcd04d577), originally named it `gil-contention` to be the same as the uses of `gil_contention` variables in `SystemMonitor` but have no strong opinion about keeping it that way. :+1: